LICENSE

```diff
@@ -1,16 +1,21 @@
-MIT License
+General Potato License
 
-Copyright (c) 2022 Sentry
+Copyright (c) 2022 Functional Software, Inc.
 
-Permission is hereby granted, free of charge, to any person obtaining a copy
-of this software and associated documentation files (the "Software"), to deal
-in the Software without restriction, including without limitation the rights
-to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-copies of the Software, and to permit persons to whom the Software is
-furnished to do so, subject to the following conditions:
+If you feel appreciation,
+For a colleague's dedication,
+You can use this software free—
+Gib potatoes liberally!
```

Comment:
Nitpick, but with GibPotato, the plural of potato is potato 😬

Reply:
```suggestion
Gib potato liberally!
```